app/uk/gov/hmrc/customs/declaration/controllers/CustomsDeclarationsController.scala

```diff
@@ -68,88 +70,98 @@ class CustomsDeclarationsController @Inject()(logger: DeclarationsLogger,
     case _ => Right(AnyContentAsEmpty)
   })
 
```

Comment:
I would suggest something returning something like `Either[Seq[String], Unit]` or `Option[Seq[String]]` . I think this reveals the intent better and would make the client code clearer

Reply:
String is not the best solution, we need to call XmlResult in client code at some point (line 105).

Reply:
Sorry I did not read the code properly - I meant something like `Either[Seq[ErrorResponse], Unit]` or `Option[Seq[ErrorResponse]]`